.devcontainer/postStartCommand.sh

```diff
@@ -0,0 +1,14 @@
+# Add new remote repositories for origin (forked) and upstream (original)
+#git remote add upstream https://github.com/twjudson/aata
+#git remote add origin https://github.com/jbrock1999h/aata
+
+# Reset to origin main (This is for if the codespace has not caught up to changes made previously)
+#git pull origin main
+#git reset --hard origin/main
+
+# Build pretext project
+#pretext build
```

Comment:
The pretext generate and pretext build should happen in the Docker container build, not here.